charter.md

```diff
@@ -1,52 +1,78 @@
+Steering Committee.
 
 ## Mission
 
-The role of the steering committee is to instantiate the formal process
-for Kubernetes governance. In addition to defining the governance
-process, it is important to provide a means for iterating on processes 
-defined by the steering committee.  The role of the steering committee 
-is to be a live, responsive body that can refactor and reform as necessary 
-to adapt to a changing project and community.
-
-## Scope and Objectives
-The Steering Committee has a set of rights and responsibilities including the
-following:
-
-* Decide which project subgroups are responsible for each area of the project
-  and to delegate appropriate authority those groups.
-* Define, evolve, and defend the vision, values, mission, and scope of the
-  project.
-* Charter and refine policy for defining new community groups (including
-  Special Interest Groups, Working Groups, and Committees), and establish
-  transparency and accountability policies for such groups
-* Define, evolve, and defend a Code of Conduct, which must include a neutral,
-  unbiased process for resolving conflicts
-* Define and evolve project governance structures and policies, including how
-  contributors become committers/maintainers, approvers, reviewers, members,
-  etc.
+The Kubernetes Steering Committee is the governing body of the Kubernetes
+project, providing decision-making and oversight pertaining to the Kubernetes
+project bylaws, sub-organizations, and financial planning.  The Steering 
+Committee also defines the project values and structure.
+
+## How
+
+* Adapt the role and structure of the Steering Committee as needed to meet the
+  needs of the project.
+* Responsibilities not explicitly delegated to other
+  parties<sup>[2](#footnote2)</sup> through their charters reside with
+  the Steering Committee.
+* All management<sup>[1](#footnote1)</sup> responsibilities should be delegated to other
+  parties<sup>[2](#footnote2)</sup>.
+* All technical responsibilities should be delegated to SIGs (i.e. the SC shouldn't
+  retain technical responsibilities itself).
+
+## Direct responsibilities of the Steering Committee
+
+The following responsibilities belong directly to the Steering Committee.
+
+* Through the chartering review process, delegate ownership of, responsibility for
+  and authority over areas of the project to specific entities<sup>[2](#footnote2)</sup>.
+* Define, evolve, and defend the non-technical vision / mission and the values
+  of the project.
+* Charter and refine policy for defining new community groups<sup>[3](#footnote3)</sup>,
+  and establish transparency and accountability policies for such groups
+* Define and evolve project and group<sup>[3](#footnote3)</sup> governance
+  structures and policies<sup>[4](#footnote4)</sup>.
+* Act as a final non-technical escalation point for any Kubernetes repository<sup>[5](#footnote5)</sup>.
+* Request funds and other support from the CNCF (e.g. marketing, press, etc.)
```

Comment:
@michelleN @spiffxp @dims - are there more details around requests that we want to explicitly call out here? 

IMO we should expand the scope of requests.

Reply:
decided to make this part of a separate PR on the steering call.